html/semantics/forms/form-submission-0/form-data-set-empty-file.window.js

```diff
@@ -0,0 +1,14 @@
+promise_test(() => {
+  const form = document.body.appendChild(document.createElement("form")),
+        input = form.appendChild(document.createElement("input"));
+  input.type = "file";
+  input.name = "hi";
+  const fd = new FormData(form),
+        entry = fd.get(input.name);
```

Comment:
nit: I'd like to name this variable "value" rather than "entry" because "entry" is confusing with [entry](https://xhr.spec.whatwg.org/#concept-formdata-entry)